hedera-node/cli-clients/src/main/java/module-info.java

```diff
@@ -8,4 +9,10 @@
     requires com.swirlds.platform;
     requires com.swirlds.common;
     requires info.picocli;
+    requires org.apache.logging.log4j;
+    requires com.swirlds.logging;
```

Comment:
```suggestion
```
As you don't use logging, these lines shouldn't be here.

Reply:
Removed it in the next PR for record stream sign. Thanks.